cookbooks/cdo-varnish/libraries/helpers.rb

```diff
@@ -146,13 +146,13 @@ def process_request(behavior, _)
   out = ''
   cookies = behavior[:cookies]
   out << case cookies
-    when 'all'
-      '# Allow all request cookies.'
-    when 'none'
-      'cookie.filter_except("NO_CACHE");'
-    else
-      cookies.map{ |c| extract_cookie(c)}.join + "cookie.filter_except(\"#{cookies.join(',')}\");"
-  end
+         when 'all'
+           '# Allow all request cookies.'
+         when 'none'
+           'cookie.filter_except("NO_CACHE");'
+         else
+           cookies.map{ |c| extract_cookie(c)}.join + "cookie.filter_except(\"#{cookies.join(',')}\");"
+         end
```

Comment:
For `Style/AlignParameters` we've preferred fixed indentation.  I wonder if we should use parens here keep the `case` statement at a normal indent?
```ruby
out << (
  case cookies
  when 'all'
    '# Allow all request cookies.'
  when 'none'
    'cookie.filter_except("NO_CACHE");'
  else
    cookies.map{ |c| extract_cookie(c)}.join + "cookie.filter_except(\"#{cookies.join(',')}\");"
  end
)
```
We've been using this pattern a lot in React ([example](https://github.com/code-dot-org/code-dot-org/blob/1f6b5050c3fd8700b3618a7b9f45286534e3810f/apps/src/templates/instructions/MarkdownInstructions.jsx#L94-L103)).